code/src/components/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom'
+
+import { MovieList } from '../pages/MovieList'
+import { MovieDetails } from '../pages/MovieDetails'
+import { NotFound } from '../pages/NotFound'
+import { Nav } from './Nav'
+
+export const App = () => {
+
+  return (
+    <BrowserRouter>
+    <main className='main__grid'>
+      <Nav/>
+      <Switch>
+        <Route path='/' exact>
+          <MovieList/>
+        </Route>
+        <Route path='/category/:category' exact>
+          <MovieList/>
+        </Route>
+        <Route path='/category/movie/:movieId' exact>
+          <MovieDetails />
```

Comment:
maybe the path can be simplified to movie/:movieId instead of keeping the /category. It is no longer important to show just the 'category' words when clicking on one movie.

Reply:
I had to keep the category-part since this refers to movies clicked within a specific category. When I tried removing it I get an errormessage when clicking on a movie within for example top_rated.